src/NUnitFramework/framework/Internal/Execution/ParallelWorkItemDispatcher.cs

```diff
@@ -242,6 +242,11 @@ private void RestoreQueues()
 
         private void OnEndOfShift(object sender, EventArgs ea)
         {
+            if (StartNextShift())
```

Comment:
This is a non-trivial change, should it be in this PR?

Reply:
I didn't intend it to be here - messed up separating my changes.